api/models/usersChannels.js

```diff
@@ -339,15 +332,20 @@ const approvePendingUserInChannel = async (channelId: string, userId: string): P
 // prettier-ignore
 const approvePendingUsersInChannel = async (channelId: string): Promise<DBUsersChannels> => {
   const currentCount = await db.table('usersChannels')
-    .getAll(channelId, { index: 'channelId' })
-    .filter({ isMember: true })
+    .getAll(
+      [channelId, 'member'],
```

Comment:
To clarify: this automatically handles duplication? e.g. someone is a `moderator` *and* a `member`, it should only count as one membership

Reply:
Yes, since the role index can only have one value the users that have a role of "member" are exclusive of other users who have the roles "moderator" or "owner". See my comment below: https://github.com/withspectrum/spectrum/pull/4340#discussion_r236575578